src/OpenSage.Game/Network/SkirmishGameSettings.cs

```diff
@@ -118,6 +121,30 @@ public void Persist(StatePersister reader)
             reader.PersistUInt32(ref _mapFileSize);
             reader.PersistUInt32(ref _unknownInt4);
             reader.PersistUInt32(ref _unknownInt5);
+
+            if (version >= 4)
+            {
+                ushort unknown6 = 0;
+                reader.PersistUInt16(ref unknown6);
```

Comment:
I believe the first byte here is for the `Limit Superweapons` behavior - I wasn't able to figure out the second byte though.

Reply:
Nice - https://github.com/OpenSAGE/OpenSAGE/commit/9f689a9b4d4b8e3add2bac8aae0e0668c9734a52